### ISSUE TITLE
training tip

### ISSUE THREAD

**Issue by freaad** [clovaai/ext_portrait_segmentation#10]
Do you have any tip for small object trainning.
I am dealing the small binary segmentation but it doesn';t have good result. 
Do you have any tip.

**Comment by lessw2020** [comment]
A generic tip is to use focal loss for training with small objects.   Not sure if it will work here but it will certainly help in general.

**Comment by freaad** [comment]
I have already try but not a good result....

**Comment by lessw2020** [comment]
I'm testing some other loss functions tomorrow and will let you know if any progress.  Boundary loss penalty looks really good.

**Comment by freaad** [comment]
Thank you for answering. I agree the boundary loss penalty is good idea. However my problem is quite imblance data. Background is 95 % the target object is only 5 percent in the image. My problem is only binary segmentation but I cannot get the 90% iou score. I usually get the 83%. I try this network but still no luck. I am trying to use the synchronize batch normalization. Wish me good luck.
